_config.yml

```diff
@@ -7,7 +7,7 @@ description: Website of Matthew (Brender) Broberg. Matt loves working with techn
 external-image: false
 picture: assets/images/profile.jpg
 
-url: https://mbbroberg.github.io
+baseurl: '/'
```

Comment:
Hey @ammarnajjar, I never quite knew what this syntax was used for. Can talk me through your change to `baseurl` everywhere? Thanks!

Reply:
I will try to explain it.

`baseurl` is the url on which your whole jekyll website is deployed on. For github it is default to `<username>.github.io`  
so when it is set to `/`, this means to take the default baseurl and build up on that.  
I changed it from `url` to `baseurl` so it can be changed from the build command line, because to be able to be served from a docker container, the `baseurl` should be set to `localhost` not `mbbroberg.github.io`  

please refer the Dockerfile build command, or you can run in terminal (I have shortened the output here):

```bash
$ bundle exec jekyll build --help

Usage:
  jekyll build [options]

Options:
        -b, --baseurl URL  Serve the website from the given base URL
```

There is also the official docs:
https://jekyllrb.com/docs/configuration/options/#serve-command-options